app/models/user.rb

```diff
@@ -17,4 +19,8 @@ class User < ApplicationRecord
   def convert_email_to_downcase
```

Comment:
why we need to converted it? If for validation reason we have  `case_sensitive: false` in the case of uniqueness and you have `i` in the end of regex.